src/Navbar.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export default function Navbar() {
+    return (
+        <nav id="navbar-container" className="flex justify-between bg-teal-900 py-3 px-2">
```

Comment:
Can you verify this is the same color as on the design?
Here's a link on custom colors for Tailwind: https://tailwindcss.com/docs/customizing-colors 
Design Reference: https://xd.adobe.com/view/3da03753-f1e6-4bac-82db-20f1fcb93155-68f3/specs/

Reply:
Oops, I meant to leave a comment about this; I haven't figure out yet how to customize colors with Tailwind, so I just used an approximate color as a placeholder. I'll get started on customizing the colors and font :)